<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 22pt &quot;Monotype Corsiva&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>46</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>252</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;Bell MT&quot;;
color: rgb(0, 0 , 0);</string>
   </property>
   <property name="text">
    <string>Bonjour Monsieur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ticket(s)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>0</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>0</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_usr">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Créer utilisateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_usr_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Se déconnecter</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>251</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>571</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 119, 39);
font: 900 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Clore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 119, 39);
font: 900 16pt &quot;Arial Black&quot;;
</string>
   </property>
   <property name="text">
    <string>Transférer</string>
   </property>
  </widget>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>create_usr</zorder>
  <zorder>create_usr_2</zorder>
  <zorder>listView</zorder>
  <zorder>textEdit</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
